### PR TITLE
ops: fund predeploy deployer accounts for local deployment

### PR DESCRIPTION
**Pull request by tynes**
**Description**

This will make it easier to test the e2e nft deployment scripts.

### CODE REVIEW

**Review by maurelian — APPROVED** [review]

File: packages/contracts/tasks/take-dump.ts [hunk 1]
```diff
@@ -128,6 +129,40 @@ task('take-dump').setAction(async (args, hre) => {
     commit = '0000000000000000000000000000000000000000'
   }
 
+  // Insert prefunded accounts into the state dump when running on the local
+  // network
+  if (hre.network.name === 'local') {
+    const accounts = [
```
Comment (smartcontracts) [inline packages/contracts/tasks/take-dump.ts:135]: Shouldn't https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts/deploy/016-fund-accounts.ts be doing this?

Comment (smartcontracts) [reply packages/contracts/tasks/take-dump.ts:135]: Also you can get these addresses like so: https://github.com/ethereum-optimism/optimism/blob/23ed59b2dacd32227fb6a7382b53e1a8f30e6a73/packages/contracts/deploy/016-fund-accounts.ts#L27

Comment (tynes) [reply packages/contracts/tasks/take-dump.ts:135]: I can double check

**Comment by tynes** [comment]
@smartcontracts I've addressed your review, this is ready for review again

**Comment by tynes** [comment]
https://github.com/ethereum-optimism/optimism/pull/3570 depends on this PR

**Review by smartcontracts — APPROVED** [review]

**Pull request merged**